app/build.gradle

```diff
@@ -21,6 +21,14 @@ android {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
+    }
+	splits {
```

Comment:
I would switch to spaces here, looks like a rogue tab jumped in.

Reply:
this error happening on andorid emulator to install the apk